scripts/salt-install.sh

```diff
@@ -36,6 +36,19 @@ function install_salt_with_pip3() {
 function install_with_yum() {
   update_yum_repos
 
+  if [ "${OS_TYPE}" == "redhat8" ] ; then
+    yum install -y redhat-lsb-core
+    cp /tmp/repos/jumpgate-gpg-key.pub /etc/pki/rpm-gpg/jumpgate-gpg-key.pub
```

Comment:
These GPG keys have nothing to do with Python installation - why are they here? IMO these should be done via Salt before the actual RPM installation. E.g. jumpgate GPG key should be added in /saltstack/base/salt/ccmv2/init.sls, etc.

Reply:
moved to ccmv2/init.sls and prerequisites/init.sls